cuML/src/dbscan/vertexdeg/algo.h

```diff
@@ -87,6 +93,9 @@ void launcher(Pack<value_t> data, cudaStream_t stream, int startVertexId, int ba
 	 );
 
     CUDA_CHECK(cudaDeviceSynchronize());
```

Comment:
@cjnolet I understand that you didn't write this code. But please remove this synchronize call.